src/NuGet.Clients/PackageManagement.VisualStudio/Strings.resx

```diff
@@ -236,4 +236,12 @@
   <data name="CredentialProviderFailed_ImportedProvider" xml:space="preserve">
     <value>Failed to load credential provider from assembly {0}.</value>
   </data>
-</root>
+  <data name="Error_MultipleFrameworksUwp" xml:space="preserve">
+    <value>Failed to migrate project '{0}' as there are multiple target frameworks in project.json file.</value>
+    <comment>{0} is the full path to the project.</comment>
+  </data>
+  <data name="Error_TargetFrameworkNotUwp" xml:space="preserve">
+    <value>Failed to migrate dependencies for project '{0}' as they target a framework other than UAP</value>
```

Comment:
suggestion: Failed to migrate dependencies for project '{0}'. The project does not contain the UAP target framework.